scripts/analytics/download_count_summary.py

```diff
@@ -0,0 +1,54 @@
+import pytz
+
+import logging
+import requests
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from django.db.models import Q
+from website.app import init_app
+from scripts.analytics.base import SummaryAnalytics
+from osf.models import PageCounter
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
Doesn't seem like anything is being logged in this summary, so this and the `logger` imports can probably be removed. Other unused imports include `requests` and `Q`